Bigtable/src/Connection/ConnectionInterface.php

```diff
@@ -142,6 +147,16 @@ public function modifyColumnFamilies(array $args);
      */
     public function dropRowRange(array $args);
 
+    /**
+     * @param array $args
+     */
+    public function checkConsistency(array $args);
+
+    /**
+     * @param array $args
+     */
+    public function generateConsistencyToken(array $args);
+
```

Comment:
`generateConsistencyToken` + `checkConsistency` work together to poll Bigtable to see if replication is complete, which was intended to be implemented via `waitForReplication`.  If we add these two methods, please remove `waitForReplication`.